src/Init/Control/Basic.lean

```diff
@@ -20,8 +20,21 @@ def Functor.discard {f : Type u → Type v} {α : Type u} [Functor f] (x : f α)
 
 export Functor (discard)
 
+/--
+An `Alternative` functor is an `Applicative` functor that has a notion of “emptyness” or “failure”
```

Comment:
Spelling:
```suggestion
An `Alternative` functor is an `Applicative` functor that has a notion of “emptiness” or “failure”
```

But "has a notion" in my ears means "suddenly wants to". What about:
```suggestion
An `Alternative` functor is an `Applicative` functor that can "fail" or be "empty"
```